src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27377.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27377 : _IssuesUITest
+	{
+		public Issue27377(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "SwipeView: SwipeItem.IconImageSource.FontImageSource color value not honored";
+
+		[Test]
+		[Category(UITestCategories.SwipeView)]
+		public void FontImageSourceShouldHonorColor()
+		{
+			App.WaitForElement("Button");
+			App.Click("Button");
+			VerifyScreenshot();
```

Comment:
Pending snapshots. Already available in the latest build.
![image](https://github.com/user-attachments/assets/f4029bf1-f237-4b35-abcb-e640bf83d9a7)
Could you commit the images?